WorldPainter/WPCore/src/main/java/org/pepsoft/minecraft/datapack/Dimension.java

```diff
@@ -26,4 +26,7 @@
     @Builder.Default boolean hasSkylight = true;
     boolean hasCeiling;
     boolean ultrawarm;
+    public int getHeight() {
```

Comment:
No need to add a getter; Lombok does that already. Perhaps you need to install a Lombok plugin for your IDE.